clang/test/SemaOpenCL/builtins-amdgcn-raw-buffer-store-error.cl

```diff
@@ -3,12 +3,12 @@
 
 #pragma OPENCL EXTENSION cl_khr_fp16 : enable
 
-typedef char i8;
-typedef short i16;
-typedef int i32;
-typedef int i64 __attribute__((ext_vector_type(2)));
-typedef int i96 __attribute__((ext_vector_type(3)));
-typedef int i128 __attribute__((ext_vector_type(4)));
+typedef unsigned char i8;
+typedef unsigned short i16;
+typedef unsigned int i32;
+typedef unsigned int i64 __attribute__((ext_vector_type(2)));
+typedef unsigned int i96 __attribute__((ext_vector_type(3)));
+typedef unsigned int i128 __attribute__((ext_vector_type(4)));
```

Comment:
calling these vectors iN is misleading but I guess it doesn't actually matter

Reply:
That is because we said to use `<N x i32>` for consistency when the builtins were introduced.

Reply:
Yes but you can call the typedefs v2u32 or whatever, not name them something scalar-like